inception-project/inception
# #4379 - Suppress or mark concepts in the search that are marked as owl:deprecated

**What's in the PR**
- Add a new configurable IRI for a property that - when present - marks a concept, property or instance as deprecated

**How to test manually**
* Import e.g. the HPO ontology using the new HPO local profile
* On the knowledge-base page search for "obsolete" - that should mostly (only) return items that are deprecated and they should be marked as such

**Automatic testing**

**Documentation**
